configs/clip/README.md

```diff
@@ -0,0 +1,62 @@
+| vit_base_patch16_clip_384.openai_ft_in12k_in1k  |     86.57 |    49.37 |     86.87 |     98.05 | vit-base-p16_pt-64xb64_in1k-384 | vit_base_patch16_clip_384.openai_ft_in12k_in1k.bin  |
+| vit_base_patch32_clip_384.openai_ft_in12k_in1k  |     88.22 |    12.66 |     85.13 |     97.42 | vit-base-p32_pt-64xb64_in1k-384 | vit_base_patch32_clip_384.openai_ft_in12k_in1k.bin  |
+| vit_base_patch16_clip_224.laion2b_ft_in1k       |     86.57 |    16.86 |     85.49 |     97.59 | vit-base-p16_pt-64xb64_in1k-224 | vit_base_patch16_clip_224.laion2b_ft_in1k.bin       |
+| vit_base_patch32_clip_224.laion2b_ft_in12k_in1k |     88.22 |     4.36 |     83.06 |     96.49 | vit-base-p32_pt-64xb64_in1k-224 | vit_base_patch32_clip_224.laion2b_ft_in12k_in1k.bin |
+| vit_base_patch16_clip_224.laion2b_ft_in12k_in1k |     86.57 |    16.86 |     86.02 |     97.76 | vit-base-p16_pt-64xb64_in1k-224 | vit_base_patch16_clip_224.laion2b_ft_in12k_in1k.bin |
+| vit_base_patch16_clip_224.openai_ft_in1k        |     86.57 |    16.86 |      85.3 |      97.5 | vit-base-p16_pt-64xb64_in1k-224 | vit_base_patch16_clip_224.openai_ft_in1k.bin        |
+| vit_base_patch32_clip_224.openai_ft_in1k        |     88.22 |     4.36 |     81.77 |     95.89 | vit-base-p32_pt-64xb64_in1k-224 | vit_base_patch32_clip_224.openai_ft_in1k.bin        |
+| vit_base_patch16_clip_384.laion2b_ft_in1k       |     86.57 |    49.37 |     86.52 |     97.97 | vit-base-p16_pt-64xb64_in1k-384 | vit_base_patch16_clip_384.laion2b_ft_in1k.bin       |
+| vit_base_patch32_clip_384.laion2b_ft_in12k_in1k |     88.22 |    12.66 |     85.39 |     97.67 | vit-base-p32_pt-64xb64_in1k-384 | vit_base_patch32_clip_384.laion2b_ft_in12k_in1k.bin |
+| vit_base_patch16_clip_384.openai_ft_in1k        |     86.57 |    49.37 |     86.25 |      97.9 | vit-base-p16_pt-64xb64_in1k-384 | vit_base_patch16_clip_384.openai_ft_in1k.bin        |
+| vit_base_patch32_clip_448.laion2b_ft_in12k_in1k |     88.22 |     17.2 |     85.76 |     97.63 | vit-base-p32_pt-64xb64_in1k-448 | vit_base_patch32_clip_448.laion2b_ft_in12k_in1k.bin |
+| vit_base_patch16_clip_384.laion2b_ft_in12k_in1k |     86.57 |    49.37 |     87.17 |     98.02 | vit-base-p16_pt-64xb64_in1k-384 | vit_base_patch16_clip_384.laion2b_ft_in12k_in1k.bin |
+| vit_base_patch16_clip_224.openai_ft_in12k_in1k  |     86.57 |    16.86 |     85.99 |     97.72 | vit-base-p16_pt-64xb64_in1k-224 | vit_base_patch16_clip_224.openai_ft_in12k_in1k.bin  |
+| vit_base_patch32_clip_224.laion2b_ft_in1k       |     88.22 |     4.36 |     82.46 |     96.12 | vit-base-p32_pt-64xb64_in1k-224 | vit_base_patch32_clip_224.laion2b_ft_in1k.bin       |
+
+*Models with * are converted from the [official repo](https://github.com/rwightman/pytorch-image-models). The config files of these models are only for inference. We don't ensure these config files' training accuracy and welcome you to contribute your reproduction results.*
+
+## Citation
+
+```bibtex
+@misc{https://doi.org/10.48550/arxiv.2103.00020,
```

Comment:
@InProceedings{pmlr-v139-radford21a,
  title = 	 {Learning Transferable Visual Models From Natural Language Supervision},
  author =       {Radford, Alec and Kim, Jong Wook and Hallacy, Chris and Ramesh, Aditya and Goh, Gabriel and Agarwal, Sandhini and Sastry, Girish and Askell, Amanda and Mishkin, Pamela and Clark, Jack and Krueger, Gretchen and Sutskever, Ilya},
  booktitle = 	 {Proceedings of the 38th International Conference on Machine Learning},
  year = 	 {2021},
  series = 	 {Proceedings of Machine Learning Research},
  publisher =    {PMLR},
}